### PR TITLE
test: Add CCV Test Suite with Gaia provider

### PR DESCRIPTION
**Pull request by glnro**
# Description

This PR imports the working version of Gaia's [Rho release](https://github.com/cosmos/gaia/issues/1909) and introduces a new `CCVTestSuite` with Gaia as the provider chain

# Linked issues

Closes:  #131 

Additionally Fulfills Testing for [Gaia Rho Integration](https://github.com/cosmos/gaia/pull/1938)

# Type of change

- [X] Testing

# How was the feature tested?

- [X] E2E tests

### CODE REVIEW

File: app/consumer-democracy/app.go [hunk 1]
```diff
@@ -306,11 +306,11 @@ func New(
 	)
 
 	// Remove the fee-pool from the group of blocked recipient addresses in bank
-	// this is required for the provider chain to be able to receive tokens from
-	// the consumer chain
+	// this is required for the provider chain to be able to send tokens to
+	// the provider chain
```
Comment (mpoke) [inline app/consumer-democracy/app.go:310]:
```suggestion
	// this is required for the consumer chain to be able to send tokens to
	// the provider chain
```

File: app/consumer/app.go [hunk 2]
```diff
@@ -270,11 +270,11 @@ func New(
 	)
 
 	// Remove the fee-pool from the group of blocked recipient addresses in bank
-	// this is required for the provider chain to be able to receive tokens from
-	// the consumer chain
+	// this is required for the provider chain to be able to send tokens to
+	// the provider chain
```
Comment (mpoke) [inline app/consumer/app.go:274]:
```suggestion
	// this is required for the consumer chain to be able to send tokens to
	// the provider chain
```

**Review by mpoke — APPROVED** [review]
LGTM. Just some nits.

**Review by sainoe — APPROVED** [review]

**Review by smarshall-spitzbart — APPROVED** [review]
Nice

**Pull request merged**